# CellInfo Message

std_msgs/Header header

std_msgs/UInt64 timestamp
std_msgs/String repr



# struct CellInfo {
#   timestamp @0 :UInt64;
#   repr @1 :Text; # android toString() for now
# }
